### DISCUSSION TITLE
Adding theme switcher

### DISCUSSION THREAD

**Discussion by niktek** [skeletonlabs/skeleton#65] [Ideas]
To help out with testing as well as to show off the theming capabilities it would be great to add this to the docs site even though it may not be the most used component on end developer sites.

**Comment by endigo9740** [comment]
Definitely something I'd like to see happen as well. It could provide a number of benefits to testing. Plus being able to see a generated theme preview immediately would be great as well. This could make it easier to create additional curated themes for folks.

May be possible to do with Svelte as-is, which is always my preference, however I'd be open to a library such as this if it's only for the Docs site:
https://github.com/saadeghi/theme-change
